xarray/core/formatting.py

```diff
@@ -140,7 +140,7 @@ def format_item(x, timedelta_format=None, quote_strings=True):
         return format_timedelta(x, timedelta_format=timedelta_format)
     elif isinstance(x, (str, bytes)):
         return repr(x) if quote_strings else x
-    elif isinstance(x, (float, np.float)):
+    elif isinstance(x, (float, np.float_)):
```

Comment:
we might change this to just
```suggestion
    elif isinstance(x, float):
```
since
```python
In [1]: import numpy as np 
   ...:  
   ...: a = np.float64(0) 
   ...: a.dtype == float, isinstance(a, float)
Out[1]: (True, True)
```
but I'm not sure if this also works with the minimum supported numpy version (1.15 / 1.16)